### DISCUSSION TITLE
Гардероб

### DISCUSSION THREAD

**Discussion by NosokSudby** [octothorp-team/public-dobrograd#794] [Предложения]
Предлагаю добавить гардеробы на сервер, куда можно будет складывать свою одежду. 
Как по мне это добавит реализма в игру.

**Comment by Kyper1** [comment]
**Против**

*Ненужный механ. Может ещё добавить: шкафчики для оружия, коробочку для отмычек, холодильник для еды и напитков, чемодан для инструментов?*

**Reply by Lesopoval** [reply]
Это и так можно сделать.
Если у тебя есть привелегия "Ролевик", как минимум, то ты можешь использовать тулы инвентаря, ставя их на любые пропы.
